### DISCUSSION TITLE
Multiple environments for vercel deployments

### DISCUSSION THREAD

**Discussion by tvojtech** [vercel/vercel#5693] [Help] [answered]
Hello,

my backend is on Heroku and it supports three types of deployment. Development (deploying latest development code), Staging (candidate for next production deployment) and Production. Is it possible to do the  same on Vercel? Vercel reserves `development` env to local development (something else than Heroku) and deployments in cloud are only preview (staging) and live (production). Is it possible to create any other env to match env of my backend on Heroku?

**Comment by styfle** [comment, marked as answer]
Update: you can configure many Preview environments, as long as each has a corresponding git branch.

See this guide: https://vercel.com/guides/set-up-a-staging-environment-on-vercel

**Reply by styfle** [reply]
Hello @tvojtech 

Just following up now that you can configure environments per git branch

Take a look at this announcement: https://vercel.com/changelog/environments-variables-per-git-branch
